### ISSUE TITLE
Setup Board

### ISSUE THREAD

**Issue by jacola** [jacola/js2048#5]
**Board initialization**
* Create a function to initialize the board's data.
* The board array (rows) will hold arrays for each column in that row.
* Initialize the array based on the grid size to create a square grid with each element initialized to 0.
* Create a function that creates a random 2 or 4 (with equal probability) and puts it in a blank (0) spot on the board.

**Board screen elements** 
* This is done once the board is initialized and should be a new function that creates the board.
* Add a container object to store the board which we can access later.
  * Center the container using the game config's width and height. Shift it down an additional 40 pixels to account for the fact that the board is taller than it is wide.
  * Do not try to set the origin of the container!
* Add a rectangle for the board background
  * color 0xbbada0.
  * The size is the tile size + gap times grid size. Don't forget to add an additional gap so we have a margin on the right and bottom. Since the width and the height are the same, it might be good to define a constant that calculates this only once.
  * Add the board background to the container.
* Create the tiles
  * Each tile's background should be colored 0xcdc1b3.
  * It should be added to the container object.
  * Each tile needs a negative offset of 1/2 the height and the width of the board size.
* Add guide text at the bottom of the screen (y = 530)
  * message: "HOW TO PLAY: Use your arrow keys to move tiles. When two tiles with the same number touch, they merge into one!"
  * font: bold 18px sans-serif
  * color: 837c71
  * Set the origin to the top middle and the text should be centered in the game scene.
  * The wordwrap width should be 400px.
